presto-hive/src/main/java/io/prestosql/plugin/hive/HiveViewNotSupportedException.java

```diff
@@ -1,41 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.prestosql.plugin.hive;
-
-import io.prestosql.spi.connector.NotFoundException;
-import io.prestosql.spi.connector.SchemaTableName;
-
-import static java.lang.String.format;
-
-public class HiveViewNotSupportedException
```

Comment:
This means we do not have a kill-switch for Coral support.
Is it intentional?

@electrum @laurachenyu @kokosing or @martint do you happen to know?

Reply:
I intentionally remove HiveViewNotSupportedException in:
presto-hive/src/test/java/io/prestosql/plugin/hive/AbstractTestHive.java
presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java
Because these tests are no longer needed with coral integration #4661.

Reply:
> we do not have a kill-switch for Coral support.

What about `hive.translate-hive-views`?

Reply:
@kokosing good point